site/src/components/Footer.js

```diff
@@ -0,0 +1,177 @@
+//modules imports
+import React from "react";
+import styled from "styled-components";
+
+//mui import
+import TextField from "@mui/material/TextField";
+import Button from "@mui/material/Button";
+import { makeStyles } from "@material-ui/core/styles";
```

Comment:
No need for mui at all. We can have a Button component instead, similar to [this](https://github.com/layer5io/layer5/tree/master/src/reusecore/Button)